setup.cfg

```diff
@@ -0,0 +1,2 @@
+[bdist_wheel]
+universal=1
```

Comment:
Can this be specified as part of setup.py or does it have to be in a cfg? Mostly asking since everything else is in there currently having just 1 aspect in a .cfg doesn't seem ideal.